roles/edpm_podman/tasks/purge.yml

```diff
@@ -17,10 +17,50 @@
 
 - name: Clean podman images
   become: true
-  ansible.builtin.command: podman image prune -a -f
-  register: clean_podman_image
-  changed_when: clean_podman_image.rc == 0
-  failed_when: clean_podman_image.rc != 0
+  block:
```

Comment:
I don't like the approach when we need to replace "podman image prune" with own script that nobody understands. If service is temporarily down, operator should reconsile to working state. If no image is found the image will be downloaded from repository and pod will be started with this image